### PR TITLE
Remove arm/v5/v6 from build.yml

### PR DESCRIPTION
**Pull request by 99**
Removing arm/v5/v6 support from build.yml
Succesful run[ is here ](https://github.com/hashicorp/consul-terraform-sync/actions/runs/1211431864)

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -202,13 +193,13 @@ jobs:
 
       - uses: actions/upload-artifact@v2
         with:
-          name: ${{ github.event.repository.name }}_${{ needs.get-product-version.outputs.product-version }}_${{ env.FILE_SUFFIX }}.deb
-          path: out/${{ github.event.repository.name }}_${{ needs.get-product-version.outputs.product-version }}_${{ env.FILE_SUFFIX }}.deb
+          name: ${{ github.event.repository.name }}_${{ needs.get-product-version.outputs.product-version }}_amd64.deb
```
Comment (sarahethompson) [inline .github/workflows/build.yml:196]:
```suggestion
          name: ${{ github.event.repository.name }}_${{ needs.get-product-version.outputs.product-version }}_armhf.deb
```

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -202,13 +193,13 @@ jobs:
...
+          path: out/${{ github.event.repository.name }}_${{ needs.get-product-version.outputs.product-version }}_amd64.deb
```
Comment (sarahethompson) [inline .github/workflows/build.yml:197]:
```suggestion
          path: out/${{ github.event.repository.name }}_${{ needs.get-product-version.outputs.product-version }}_armhf.deb
```

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -202,13 +193,13 @@ jobs:
...
 
       - uses: actions/upload-artifact@v2
         with:
-          name: ${{ github.event.repository.name }}-${{ needs.get-product-version.outputs.product-version }}.${{ env.FILE_SUFFIX }}.rpm
-          path: out/${{ github.event.repository.name }}-${{ needs.get-product-version.outputs.product-version }}.${{ env.FILE_SUFFIX }}.rpm
+          name: ${{ github.event.repository.name }}-${{ needs.get-product-version.outputs.product-version }}.x86_64.rpm
+          path: out/${{ github.event.repository.name }}-${{ needs.get-product-version.outputs.product-version }}.x86_64.rpm
```
Comment (sarahethompson) [inline .github/workflows/build.yml:202]:
```suggestion
          path: out/${{ github.event.repository.name }}-${{ needs.get-product-version.outputs.product-version }}.${{ matrix.goarch }}.rpm
```

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -202,13 +193,13 @@ jobs:
...
-          name: ${{ github.event.repository.name }}_${{ needs.get-product-version.outputs.product-version }}_${{ env.FILE_SUFFIX }}.deb
-          path: out/${{ github.event.repository.name }}_${{ needs.get-product-version.outputs.product-version }}_${{ env.FILE_SUFFIX }}.deb
+          name: ${{ github.event.repository.name }}_${{ needs.get-product-version.outputs.product-version }}_amd64.deb
+          path: out/${{ github.event.repository.name }}_${{ needs.get-product-version.outputs.product-version }}_amd64.deb
 
       - uses: actions/upload-artifact@v2
         with:
-          name: ${{ github.event.repository.name }}-${{ needs.get-product-version.outputs.product-version }}.${{ env.FILE_SUFFIX }}.rpm
-          path: out/${{ github.event.repository.name }}-${{ needs.get-product-version.outputs.product-version }}.${{ env.FILE_SUFFIX }}.rpm
+          name: ${{ github.event.repository.name }}-${{ needs.get-product-version.outputs.product-version }}.x86_64.rpm
```
Comment (sarahethompson) [inline .github/workflows/build.yml:201]:
```suggestion
          name: ${{ github.event.repository.name }}-${{ needs.get-product-version.outputs.product-version }}.${{ matrix.goarch }}.rpm
```

**Review by sarahethompson — CHANGES_REQUESTED** [review]
Noticed the arm linux packages weren't being uploaded (added some suggestions).
I think this [line](https://github.com/hashicorp/consul-terraform-sync/blob/474c200e144d88028dc59d3b02f458e36929543a/.github/workflows/build.yml#L185) needs updated to `arch: ${{ matrix.goarch }}`

**Review by sarahethompson — APPROVED** [review]

**Review by mkam — APPROVED** [review]
LGTM, though could the commits be squashed when merged to our master branch?

**Pull request merged**